frontend/sportsmatch-app/src/styles/Match.css

```diff
@@ -32,7 +34,7 @@
 
 .match h1 {
   font-family: var(--sm-secondary-font);
-  font-size: 2.5em !important;
+  font-size: var(--sm-h1-heading);
```

Comment:
are you sure removing the !important doesn't break something

Reply:
I double checked and only `Match.tsx` uses the match h1 class so it shouldn't break anything on this page or the other pages